server/test/controllers/document.specs.js

```diff
@@ -0,0 +1,373 @@
+
+import { expect } from 'chai';
+import supertest from 'supertest';
+import 'babel-register';
+import auth from '../../helper/auth';
+import mockData from '../mockData/mockData';
+import app from '../../../build/server';
+
+const superAdminToken = auth.setUserToken(mockData.superAdmin);
+const badToken = mockData.badToken;
+
+const userToken = auth.setUserToken(mockData.user);
+const request = supertest(app);
+
+describe(' Document', () => {
+  describe('create documents function', () => {
+    it('should return a message document created successfully', (done) => {
+      request.post('/api/v1/documents')
+      .send({ title: 'johnDoe@yahoo.com',
+        content: 'humanity',
+        access: 'public' }
+      )
+      .set({ Authorization: `${userToken}` })
+      .end((err, res) => {
+        expect(res.body.message).to.be.equal('New Document created successfully');
+        expect(res.statusCode).to.be.equal(201);
+        done();
+      });
+    });
+
+    it(`should return error message with status code 400 for invalid
+      access condition`, (done) => {
+      request.post('/api/v1/documents')
+      .send({ title: 'johnDoe@yahoo.com',
+        content: 'humanity',
+        access: 'publics' }
+      )
+      .set({ Authorization: `${userToken}` })
+      .end((err, res) => {
+        expect(res.body.errorMessage).to.be.equal('Invalid access parameter');
+        expect(res.statusCode).to.be.equal(400);
+        done();
+      });
+    });
+
+    it(`should return error message with status code 400 if the document title
+      already exist`, (done) => {
+      request.post('/api/v1/documents')
+      .send({ title: 'Django lady',
+        content: 'duplicate',
+        access: 'public' }
+      )
+      .set({ Authorization: `${userToken}` })
+      .end((err, res) => {
+        expect(res.statusCode).to.be.equal(400);
+        expect(res.body.errorMessage).to.be.equal('Document already exist');
+        done();
+      });
+    });
+
+    it(`should return error message with status code 412 when user tries
+      to create a document with no content`, (done) => {
+      request.post('/api/v1/documents')
+      .send({ title: 'johnDoe@yahoo.com', access: 'public' })
+      .set({ Authorization: `${userToken}` })
+      .end((err, res) => {
+        expect(res.statusCode).to.be.equal(412);
+        done();
```

Comment:
Your test spec says you are testing for an error message but your assertion does not provide for that check. Why is that so?